executor_manager/services/sandbox/manager.py

```
@@ -537,6 +537,83 @@ async def cleanup_stale_sandboxes(
+                        "mode": "task_id",
+                        "task_id": sandbox_id,
+                        "result": fallback_delete_status,
+                    }
+                )
+                delete_status = fallback_delete_status
+            else:
+                delete_status = direct_delete_status
+
+            result["delete_result"] = delete_status
+            result["deleted"] = delete_status.get("status") == "success"
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)